railib/rest.py

```diff
@@ -63,15 +63,15 @@ def _contains_insensitive(items: list, term: str) -> bool:
 
 
 # Fill in any missing headers.
-def _default_headers(url: str, headers: dict = None) -> None:
+def _default_headers(url: str, headers: dict = None) -> dict:
```

Comment:
good catch, but i would fix this by changing contains_insensitive to operate on a dict.

Reply:
Right, I did that in the first commit https://github.com/RelationalAI/rai-sdk-python/pull/35/commits/61ece902b2812fcd4c5ece9a3f0ed28337f20d48. Then thought it'd make more sense for `_contains_insensitive` to accept a list. I'll change it back.

Reply:
thanks, ill go ahead an approve assuming you will make that fix. this is a pretty bad bug on my part.

Reply:
done